src/main/java/task/Command.java

```diff
@@ -0,0 +1,41 @@
+package task;
+
+public enum Command {
```

Comment:
slightly confusing if you use Command here, but then label todo deadline event and none as not command for their isCommand boolean. But good attempt nonetheless!